tests/crypto/tests.c

```diff
@@ -12,7 +12,9 @@ void TestAll()
     TestRSA();
 #endif
     TestRandom();
+#if defined(__x86_64__) || defined(__i386__)
```

Comment:
Why is this #if test needed?  At least add a comment into the code explaining it.